deploy-hub-configs/deploy.sh

```diff
@@ -187,6 +191,10 @@ EOF
     oc --kubeconfig=${KUBECONFIG_HUB} apply -f 04-agent-service-config.yml
     oc --kubeconfig=${KUBECONFIG_HUB} apply -f 05-pullsecrethub.yml
 
+    if [[ "${HAS_HTTPD}" -eq "0" ]]; then
+      oc patch AgentServiceConfig agent --type=json -p="[{'op': 'remove', 'path': '/spec/osImages'}]"
```

Comment:
Don't you need to have `osImages` configured? Or is the intention that this would pull from upstream (connected)?

Reply:
Yes, omitting osImages causes it to pull some default images from mirror.openshift.com